webpack-helper/configs.coffee

```diff
@@ -88,7 +88,7 @@ optionConfigs =
     ]
     devServer:
       contentBase: './'
-      publicPath: "http://localhost:#{DEV_PORT}/dist/"
+      publicPath: "http://localhost:#{DEV_PORT}/assets/"
       historyApiFallback: true
```

Comment:
We should chat about this, I've implemented it a different way in https://github.com/openstax/concept-coach/pull/3,  I extended the build to set the baseUrl based on an ENV var.